### ISSUE TITLE
Consider adding additional versions

### ISSUE THREAD

**Issue by SerLizar** [nginxinc/docker-nginx#637]
Other docker official images I've seen have a "standard" image using debian:bullseye, a "slim" variation using debian:bullseye-slim and an "alpine" using the latest alpine:<mayor>.<minor> is when the image is made.

This one is missing the "standard" debian:bullseye, which makes it so that the power of layers is either not used (by people using the "standard" version of other official images, not knowing how layers work and not looking into it) or forcing people to use the "slim" or "alpine" version of their other images so that they can take advantage of them.

**Comment by thresheek** [comment]
What would the advantage of having an nginx image on top of "fat" bullseye image be?  As far as I can see, relevant images like php also use bullseye-slim variant, so the layering system is still re-used.

**Comment by SerLizar** [comment]
The problem with that is "relevant images", which images are relevant is not up to me or you to decide, it's up to the final user.

The moment you environment requieres a single "fat" image you take the full cost of another debian layer that can't reuse the contents of the slim variant do to image flattening at that level.

Of course if you want to optimize for disk space, having as few "fat" images as possible would be desirable. However, if you are optimizing for memory usage, the moment you need at least one, you should opt for all of them to be the "fat" variant.

In short, the advantage is end user choice.

**Comment by thresheek** [comment]
What are the memory usage differences between "slim" and "fat" images?

There is also another consideration: an increased attack surface using the "fat" images.  Less software we have in the images means less possible CVEs to fix.

**Comment by thresheek** [comment]
Closing, will not implement additional "fat" images.
